lib/ansible/module_utils/facts/hardware/linux.py

```diff
@@ -454,7 +454,7 @@ def get_mount_facts(self):
         for fields in mtab_entries:
             device, mount, fstype, options = fields[0], fields[1], fields[2], fields[3]
 
-            if not device.startswith('/') and ':/' not in device:
+            if not device.startswith('/') and ':/' not in device and not fstype.startswith('fuse'):
```

Comment:
Rather than continuing to make this conditional more complex, I suggest making a filter function to encapsulate this logic. Then the list of `mtab_entries` can be filtered before looping.

```python
filtered_mtab_entries = (e for e in mtab_entries if _filter_mtab_entries(e))
```

Or the `filter()` built-in could be used rather than a generator:
```python
filtered_mtab_entries = filter(_filter_mtab_entries, mtab_entries)
```